Vagrantfile

```diff
@@ -1,11 +1,19 @@
 # -*- mode: ruby -*-
 # vi: set ft=ruby :
 
-# Automated creation of testing environments on misc. platforms
+# Automated creation of testing environment and standalones of borg-backup for various platforms.
```

Comment:
This is my vagrant n00bness shining through. I had no idea of the existence
of vagrant scp. Feel free to junk the whole PR if you like.
On 29/09/2015 9:23 PM, "TW" <notifications@github.com> wrote:

> In Vagrantfile
> <https://github.com/borgbackup/borg/pull/223#discussion_r40659669>:
>
> > @@ -1,11 +1,19 @@
> >  # -*- mode: ruby -*-
> >  # vi: set ft=ruby :
> >
> > -# Automated creation of testing environments on misc. platforms
> > +# Automated creation of testing environment and standalones of borg-backup for various platforms.
>
> s/standalones of borg-backup/borgbackup standalone binaries/
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/borgbackup/borg/pull/223/files#r40659669>.
>